package.json

```diff
@@ -159,7 +160,7 @@
     ]
   },
   "scripts": {
-    "build": "webpack --mode production --entry ./src/js/main.js",
+    "build": "webpack --mode development --entry ./src/js/main.js",
```

Comment:
when working on the gui -> `npm run watch`... that will rebuild only the parts affected by your change (so you get from ~50s builds to ~5s incremental builds) while still providing linter feedback...

Reply:
what the hell man! thanks a lot! that really made my day. incredible :)